website/azurerm.erb

```diff
@@ -1600,6 +1604,10 @@
                   <a href="/docs/providers/azurerm/r/local_network_gateway.html">azurerm_local_network_gateway</a>
                 </li>
 
+                <li>
+                  <a href="/docs/providers/azurerm/r/nat_gateway.html">azurerm_nat_gateway</a>
```

Comment:
It looks like docs for the association resource are missing?

Reply:
Added